lib/miq_ldap.rb

```diff
@@ -45,9 +45,12 @@ def initialize(options = {})
     @follow_referrals = options.delete(:follow_referrals) || ::Settings.authentication.follow_referrals
     options[:host] ||= ::Settings.authentication.ldaphost
     options[:port] ||= ::Settings.authentication.ldapport
+    options[:encryption] = {}
+
     options[:encryption] = {:method => :simple_tls} if mode == "ldaps"
 
     options[:host] = resolve_host(options[:host], options[:port])
+    options[:encryption][:tls_options] = {:verify_mode => OpenSSL::SSL::VERIFY_NONE} if options[:host].ipaddress?
```

Comment:
If you're changing things... prefer `store_path`.
```ruby
options.store_path(:encryption, :tls_options, :verify_mode, OpenSSL::SSL::VERIFY_NONE) if options[:host].ipaddress?
```

Reply:
👍